src/frontend/src/test-e2e/register.test.ts

```diff
@@ -105,51 +105,123 @@ test("Register new identity and add additional device", async () => {
   });
 }, 300_000);
 
+const initDeviceLink = async (
```

Comment:
so far we've either duplicated code or added helpers to `flows.ts`. Would it make sense to stay consistent?

Reply:
sure